app/routes/funding/what-funding.js

```diff
@@ -24,35 +27,37 @@ module.exports = [{
   options: {
     validate: {
       payload: Joi.object({
-        standard: Joi.array().items(Joi.string()).single()
+        standard: Joi.required()
```

Comment:
Joi did not respond well to attempting to validate the standard payload was an array with at least 1 item in it, therefore I used required. Does mean a user could POST a payload other than an array to this endpoint most likely disrupting the code, but they would need authorisation token to do so.

Reply:
I think if you use `required()` on the array it should fix it.

ie. `Joi.array().items(Joi.string()).single().required()`

Reply:
Yup, `Joi.array().items(Joi.string()).single().required()` worked, added and tests passing